django_fsm_freeze/models.py

```diff
@@ -113,16 +117,12 @@ def config_check(cls) -> None:
     def save(self, *args, **kwargs) -> None:
         """Data freeze checking before saving the object."""
 
-        if not self._bypass_fsm_freeze:
-            if kwargs.get('force_insert', None):
-                # e.g. object creation
```

Comment:
I'd kind of like it to be more clear that "force_insert" is coming from object creation or sth (from django, i.e. outside of this lib). The proposed change looks a bit magical to me, meaning, why are we caring about "force_insert" and what is it.
Maybe add a comment for this.